docpage/docs/Introduction.md

```diff
@@ -9,12 +9,12 @@ DCNiOS is an open-source command-line tool that easily manages the creation of e
 
 ![DCNiOS images](/../static/img/dcnios-logo-hor.png)
 
-Apache NiFi Process Group is a group of Processors that compose a dataflow. DCNiOS uses predefined Process Groups that make simple actions like interacting with a third-party component (e.g., consuming from Kafka) or changing the data content (e.g.encoding the data in base64) to compose a complete dataflow. 
+Apache NiFi Process Group is a group of Processors that compose a dataflow. DCNiOS uses predefined Process Groups that make simple actions like interacting with third-party elements (e.g., consuming from Kafka) or changing the data content (e.g.encoding the data in base64) to compose a complete dataflow. 
```

Comment:
An Apache NiFi Process Group is a group of Processors that compose a dataflow. DCNiOS uses predefined Process Groups that make simple actions like interacting with third-party elements (e.g., consuming from Kafka) or modifying the data content (e.g., encoding the data in base64) to compose a complete dataflow.